IS0401Test.py

```diff
@@ -1206,6 +1206,76 @@ def test_21(self, test):
 
         return test.PASS()
 
+    def test_22(self, test):
+        """Node resource IDs persist over a reboot"""
+
+        return test.MANUAL("This check must be performed manually, or via use of the following tool",
+                           "https://github.com/AMWA-TV/nmos-testing/blob/master/utilities/uuid-checker/README.md")
+
+    def test_23(self, test):
+        """Senders and Receivers correctly use BCP-002-01 grouping syntax"""
+
+        found_groups = False
+        groups = {"node": {}, "device": {}}
+        for resource_name in ["senders", "receivers"]:
+            valid, response = self.do_request("GET", self.node_url + resource_name)
+            if valid and response.status_code == 200:

```

Comment:
Similarly, what do you think about reporting a `FAIL` here if not valid or wrong status code, though that would duplicate a failure elsewhere in the test suite?

Reply:
I think we should try to keep each test case as isolated as possible. That may well result in some cases where you get a PASS which turns to a FAIL when you fix something else, but I think that's going to be easier for us to maintain as a general rule.

Reply:
👍